terrautils/gdal.py

```diff
@@ -68,3 +69,17 @@ def centroid_from_geojson(geojson):
 def wkt_to_geojson(wkt):
     geom = ogr.CreateGeometryFromWkt(wkt)
     return geom.ExportToJson()
+
+
+def find_plots_intersect_boundingbox(bounding_box, all_plots):
+    bbox_poly = ogr.CreateGeometryFromJson(str(bounding_box))
+    intersecting_plots = []
+
+    for plotname in all_plots:
+        bounds = all_plots[plotname]
+        yaml_bounds = yaml.safe_load(bounds)
+        current_poly = ogr.CreateGeometryFromJson(str(yaml_bounds))
+        intersection = bbox_poly.Intersection(current_poly)
+        if intersection is not None:
```

Comment:
I don't know if this will work, in the example I showed you the intersection result has the value:
```
'{ "type": "GeometryCollection", "geometries": [ ] }'
```
when there is no intersection, not None. You'll need to check if geometries is in intersection and it has length zero i think.

Reply:
Sure. It comes up as "None" sometimes when there is no intersection, so I will put that inside the is not None conditional.